src/GTP.cpp

```diff
@@ -197,6 +197,7 @@ const std::string GTP::s_commands[] = {
     "lz-genmove_analyze",
     "lz-memory_report",
     "lz-setoption",
+    "avoid",
```

Comment:
Prefix the command with `lz-` (like the ones above it), because it's not part of the standard protocol (see https://www.lysator.liu.se/~gunnar/gtp/gtp2-spec-draft2/gtp2-spec.html#sec:private-extensions).